src/styles/main.scss

```diff
@@ -12,3 +12,27 @@ body {
 
 @import "_header.scss";
 @import "_catalog.scss";
+
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 488px) and (max-width: 767px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (max-width: 487px) {
+  .catalog {
+    grid-template-columns: 200px;
+  }
+}
```

Comment:
1.move it to the catalog.scss file and use scss nesting 
2. simplify yours media queries you don't need `and` keyword in this task